CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [CHANGE] Querier: returning 422 when query hits `max_fetched_chunks_per_query` and `max_fetched_series_per_query` limits in the store-gateway. #4056
 * [CHANGE] Packaging: Migrate FPM packaging solution to NFPM. Rationalize packages dependencies and add package for all binaries. #3911
+* [CHANGE] Query-frontend: Wait for in-flight queries to finish before shutting down. #4073
```

Comment:
A couple of things:
1. Not a change, but ENHANCEMENT
2. Clarify this applies when the query-frontend is used with query-scheduler

Reply:
Thanks, will do.

Reply:
Done.